oak_functions/wasm/src/lib.rs

```diff
@@ -271,7 +196,91 @@ where
     }
 }
 
+// Exports the functions from oak_functions_abi/src/lib.rs.
+// To instantiate this linker, use the same store used creating the new linter.
```

Comment:
nit: This should probalby be a doccomment, and perhaps the last part could be "use the same store used when creating the new linter."

Reply:
I refactored to also add instantiate to the OakLinker (had to fight the borrow checker a bit). Could you have a second look? Thank you!